options.js

```diff
@@ -41,20 +50,43 @@ function restore_options() {
 		document.getElementById( "font_family" ).value = values[ "font_family" ];
 		document.getElementById( "style_right" ).value = values[ "style_right" ];
 		document.getElementById( "style_top" ).value = values[ "style_top" ];
+		document.getElementById( "z_index" ).value = values[ "z_index" ];
+		domains_array = values[ "domains_array" ];
+		recreate_domains_list_in_html( domains_array );
 	});
 }
 
 document.addEventListener( 'DOMContentLoaded', restore_options );
 document.querySelector( '#save' ).addEventListener( 'click', save_options );
 
+// To let function get not Window as 'this', but exactly those element, which called function.
+document.getElementById( "add_domain" ).addEventListener( 'click', function() {
+	add_this_domain( this );
+});
+
+document.getElementById( "switch_to_page_1" ).addEventListener( 'click', function() {
+	// to hide, to show
+	switch_options_page( "page_2", "page_1" );
+	switch_options_page( "page_2_button", "page_1_button" );
+	} );
+document.getElementById( "switch_to_page_2" ).addEventListener( 'click', function() {
```

Comment:
Page 2 only displays when the configuration is in full screen. In other words it doesn't show when I click the extension button.

![image](https://github.com/yaronf/overlay-clock/assets/1265636/72958421-827f-43da-965c-9e21867c22dd)

Reply:
Yours extension is very narrow (while i have `min-page-width`). Dunno why. But i checked only in firefox with here and there changes, so who knows, does it affect it.

The strange thing is, i deleted `(0..1)` part in `background opacity`, but you still have it.

Reply:
> Yours extension is very narrow (while i have `min-page-width`). Dunno why. But i checked only in firefox with here and there changes, so who knows, does it affect it.
> 

Any idea how to make the `browser`/`chrome` code work in both browsers?

> The strange thing is, i deleted `(0..1)` part in `background opacity`, but you still have it.

Yes, sorry. I have two Chrome instances (two "profiles") and I didn't even know that each one has its own extensions.

Reply:
Oh, firefox is greater browser, so supports both ways.